api/src/routes/settings.ts

```diff
@@ -113,6 +113,52 @@ export const settingRoutes: FastifyPluginCallbackTypebox = (
     }
   );
 
+  fastify.put(
+    '/update-my-socials',
+    {
+      schema: {
+        body: Type.Object({
+          website: Type.Optional(Type.String({ format: 'url' })),
+          twitter: Type.Optional(Type.String({ format: 'url' })),
+          githubProfile: Type.Optional(Type.String({ format: 'url' })),
+          linkedin: Type.Optional(Type.String({ format: 'url' }))
+        }),
+        response: {
+          200: Type.Object({
+            message: Type.Literal('flash.updated-socials'),
+            type: Type.Literal('success')
+          }),
+          500: Type.Object({
+            message: Type.Literal('flash.wrong-updating'),
+            type: Type.Literal('danger')
+          })
+        }
+      }
+    },
+    async (req, reply) => {
+      try {
+        await fastify.prisma.user.update({
+          where: { id: req.session.user.id },
+          data: {
+            website: req.body.website,
+            twitter: req.body.twitter,
+            githubProfile: req.body.githubProfile,
+            linkedin: req.body.linkedin
+          }
```

Comment:
```suggestion
          data: {
            portfolio: {
              website: req.body.website,
              twitter: req.body.twitter,
              githubProfile: req.body.githubProfile,
              linkedin: req.body.linkedin
            }
          }
```

Should resolve the type error

Reply:
Typo, I thought they don't exist in Prisma modal, so I were waiting 🤣😅